core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -66,6 +66,7 @@ object PrimOp {
 
 abstract class Arg {
   def localName: String = name
+  def simpleName: String = name
```

Comment:
```suggestion
  private[chisel3] def simpleName: String = name
```
Let's make these added methods package private. People aren't supposed to touch these internal classes but it's a mistake that they're public anyway so let's make sure anything new is private.